connectors/src/lib/databases.ts

```diff
@@ -77,7 +77,11 @@ type UpsertRowsParams = {
   dataSourceConfig: DataSourceConfig;
   databaseId: string;
   tableId: string;
-  contents: Record<string, Record<string, string | number | boolean | null>>;
+  contents: {
+    created: number;
```

Comment:
Why is created a param ? That should be automatically set by `core` on creation no ?  it's system-ish